### ISSUE TITLE
[Feature Request] add icons to each long-press popup menu entry

### ISSUE THREAD

**Issue by ronidee** [TeamNewPipe/NewPipe#4931]
First of all I wanna say that your clean up of the popup menu is really clever. Enqueuing only exists once etc. makes the menu really less cluttered, love it! <3

Now to further increase the easy of use and speed with which users (including ME :D) can navigate this menu, I'd suggest to add icons infront of each menu item. 
I say this increase the easy of use, because users can recognize the menu entry they're searching for even quicker. Textual buttons are a good design principle in general, Gnome 3 e.g. uses them very effectively (just stating this so it doesn't seem it's only my personal opinion 😋)

Since this is a very small feature, I just wanted to make sure it's not **unwanted** and my effort wouldn't be for nothing, when I do this and create a PR.

**TL;DR**
@Devs: Would you merge a PR adding icons to each menu entry inside the long-press popup menu?
@Users: Would you like to see/have icons in the mentioned popup menu?

Cheers!

**Comment by MD77MD** [comment]
this is a must, I have seen this already been done  along time ago somewhere. but haven't been merged. maybe one of the maintainers refer to it here

**Comment by mhmdanas** [comment]
@ronidee please open an issue that follows the template from [here](https://github.com/TeamNewPipe/NewPipe/issues/new/choose).

**Comment by ronidee** [comment]
Yeah, no. Nevermind, I guess it's easier for you to do it yourself than to implement mine.
cheers!
